components/match2/wikis/trackmania/match_group_display_bracket_custom.lua

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
this module in its current state is redundant (if module info is updated and module opponentDisplay is set up correctly then the commons module grabs the correct opponent library automatically)